### ISSUE TITLE
tests/common/util.rs panics with Broken pipe error on some targets when pipe_in() is used in tests

### ISSUE THREAD

**Issue by zhitkoff** [uutils/coreutils#5338]
https://github.com/uutils/coreutils/blob/aa0437c28a07364244b43bea53f6e4fd7d100fba/tests/common/util.rs#L2014-L2015

This seems to be the culprit. Maybe add handling the errors with match/if let instead of expect().unwrap() ?

**Comment by tertsdiepraam** [comment]
Hmm, what would we then do in the error branch? I guess we _could_ ignore the error, but that shouldn't be necessary. Which targets are impacted by this?

**Comment by zhitkoff** [comment]
yes, specifically for broken pipe - ignore it. Panic on the other errors. Targets where it failed consistently are ubuntu-latest intel (i686, x84_64). All other targets did not fail on the same tests. The workaround is to use `.ignore_stdin_write_error()` of `UCommand` instances in tests that use `.pipe_in()` stdin feed, but it should not be required - it works without it on all other targets.
